build-scripts/rollup.node.test.config.js

```diff
@@ -16,7 +20,7 @@ export default function createNodeTestConfig(options = {}) {
         outro: '});'
       },
       plugins: [alias({ '@pollyjs-tests': testsPath }), multiEntry()],
-      external: ['chai']
+      external: [...pollyDependencies, 'node-fetch', 'chai']
```

Comment:
Mark all polly dependencies as external so they arent built into the test bundle